### PR TITLE
Docker container image for Repose Internal

### PR DESCRIPTION
**Pull request by deep8103**
- Identity integration tested and working fine.
- TenantId authorization is pending. 
- Tested in a containerized env with a running AtomHopper.
- To create a container for local, the only change required is in build-arg: repose_valve (covered in the docker file documentation)
Example:
`docker build --build-arg schema_version=1.137.0 --build-arg feeds_filters_version=1.7.0 --build-arg saxon_lic=saxon-license.lic -f Dockerfile -t repose-internal:1.9.0.2 . --build-arg repose_valve=internal --no-cache`

### CODE REVIEW

**Comment by deep8103** [comment]
It contains the changes of https://github.com/rackerlabs/cloudfeeds-repose-filters/pull/33 but that would not be visible after the merge of this PR.

Only files changed in this PR: 
1. docker/configuration/internal/system-model.cfg.xml 
2. docker/configuration/internal/http-connection-pool.cfg.xml

File: docker/configuration/internal/http-connection-pool.cfg.xml [hunk 1]
```diff
@@ -14,7 +14,6 @@
         http.connection.timeout="30000"
         http.connection.max-line-length="8192"
         http.connection.max-header-count="256"
-        http.connection.max-status-line-garbage="100"
```
Comment (AJStieren) [inline docker/configuration/internal/http-connection-pool.cfg.xml:17]: Is this config gone in the current filter version or was it just non-functional?

File: docker/configuration/common/keystone-v2.cfg.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            set-catalog-in-header="false" />
+    <delegating quality="0.9"/>
+    <cache>
+        <timeouts variability="10">
+            <token>600000</token>
+            <group>60000</group>
+        </timeouts>
+    </cache>
+    <!--tenant-handling send-all-tenant-ids="true" /--> 
+</keystone-v2>
```
Comment (AJStieren) [inline docker/configuration/common/keystone-v2.cfg.xml:20]: Super mega ultra nitpicking request: Add an empty line at the end of the file.
```suggestion
</keystone-v2>

```

It makes me happy. Also there is a super niche scenario of very specific systems failing to read the last line in files.

**Review by AJStieren — CHANGES_REQUESTED** [review]
Dealing with a hardware failure. Will test once I've got it... mitigated. In the meantime had a tiny request and a question.

File: docker/configuration/internal/http-connection-pool.cfg.xml (hunk 1 above)
Comment (deep8103) [reply docker/configuration/internal/http-connection-pool.cfg.xml:17]: Removed in the newer version of Repose.

File: docker/configuration/common/keystone-v2.cfg.xml (hunk 2 above)
Comment (deep8103) [reply docker/configuration/common/keystone-v2.cfg.xml]: Done.

**Review by AJStieren — APPROVED** [review]

**Pull request merged**